### PR TITLE
Search summary modal, csv feedback

### PR DESCRIPTION
**Pull request by SanjeevLakhwani**
- Added search summary modal
- Added feedback when you export a csv
- Changed the button Add conditions on datatype to + Data type

### CODE REVIEW

**Review by davidlougheed — CHANGES_REQUESTED** [review]

File: src/components/explorer/SearchSummaryModal.js [hunk 1]
```diff
@@ -1,225 +1,140 @@
-import React from "react";
-import {parse} from "iso8601-duration";
-import {Col, Divider, Modal, Row, Statistic, Typography} from "antd";
+/* eslint-disable camelcase */
```
[the remark this answers is not in the document]
Comment (SanjeevLakhwani) [reply src/components/explorer/SearchSummaryModal.js:1]: line 25: `const ids = searchResults.results.results.map(({ subject_id }) => subject_id);`

File: src/components/explorer/SearchSummaryModal.js [hunk 2]
```diff
@@ -1,225 +1,140 @@
...
+            headers: myHeaders,
+            body: raw,
+            redirect: "follow",
+        };
+
+        fetch(`${katsuUrl}/api/search_overview`, requestOptions)
+            .then((response) => response.json())
+            .then((result) => {
+                setData(result);
+                console.log("what i got", result);
```
[the remark this answers is not in the document]
Comment (SanjeevLakhwani) [reply src/components/explorer/SearchSummaryModal.js:42]: removed it

File: src/modules/explorer/actions.js [hunk 3]
```diff
@@ -2,8 +2,6 @@ import {createNetworkActionTypes, networkAction} from "../../utils/actions";
 import {jsonRequest} from "../../utils/requests";
 import {extractQueriesFromDataTypeForms} from "../../utils/search";
 
-import FileSaver from "file-saver";
```
[the remark this answers is not in the document]
Comment (SanjeevLakhwani) [reply src/modules/explorer/actions.js:5]: it is

File: src/components/explorer/SearchSummaryModal.js [hunk 4]
```diff
@@ -1,225 +1,140 @@
...
 
-    // this doesn't work, many searches incorrectly return all experiments instead of a subset
-    // const experiments = searchResults?.results?.results?.experiment || [];
+    const katsuUrl = useSelector((state) => state.services.itemsByArtifact.metadata.url);
 
-    // instead pull experiments from phenopackets
-    const experiments = searchFormattedResults.flatMap(r => r.experiments);
+    useEffect(() => {
+        const myHeaders = new Headers();
+        myHeaders.append("Content-Type", "application/json");
```
[the remark this answers is not in the document]
Comment (SanjeevLakhwani) [reply src/components/explorer/SearchSummaryModal.js:23]: changed it

**Review by davidlougheed — APPROVED** [review]

**Pull request merged**